lib/components/src/tabs/tabs.tsx

```diff
@@ -122,7 +145,7 @@ export const Tabs = React.memo(
                 type="button"
                 key={id}
                 active={active}
-                onClick={e => e.preventDefault() || actions.onSelect(id)}
+                onClick={(e: any) => e.preventDefault() || actions.onSelect(id)}
```

Comment:
Isn't this just `e:Event` if you don't want to be more specific?

Reply:
Hmm yep I missed that but know that I look at this line I don't really understand it, what is
`e => e.preventDefault() || actions.onSelect(id)` supposed to do? because `e => e.preventDefault()` is always defined 🤔 

 cc @ndelangen 😇

Reply:
`e.preventDefault()` will return `undefined` always.

so `e.preventDefault() || actions.onSelect(id)` will call both.

it's the same as

```
(e: any) => {
  e.preventDefault();
  actions.onSelect(id);
}
```